source/extensions/filters/http/jwt_authn/filter.cc

```diff
@@ -119,8 +121,15 @@ void Filter::onComplete(const Status& status) {
         status == Status::JwtAudienceNotAllowed ? Http::Code::Forbidden : Http::Code::Unauthorized;
     // return failure reason as message body
     decoder_callbacks_->sendLocalReply(
-        code, ::google::jwt_verify::getStatusString(status), nullptr, absl::nullopt,
-        generateRcDetails(::google::jwt_verify::getStatusString(status)));
+        code, ::google::jwt_verify::getStatusString(status),
+        [uri = this->original_uri_, status](Http::ResponseHeaderMap& headers) {
+          std::string value = absl::StrCat("Bearer realm=\"", uri, "\"");
+          if (status != Status::JwtMissed) {
+            absl::StrAppend(&value, ", error=\"", InvalidTokenErrorString, "\"");
```

Comment:
Maybe make `InvalidTokenErroString` contain the full `error=` string so that we don't have to concat this every time?

Reply:
We can do that, but we'll still need to append a `", "` to the value before the InvalidTokenErrorString

Reply:
Wait, scratch that. I see what you mean now.